src/util/accessibility/live.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable, Inject, OnDestroy} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+
+
+@Injectable()
+export class Live implements OnDestroy {
+  private _element: HTMLElement;
+
+  constructor(@Inject(DOCUMENT) private _document: any) { this._element = this._createElement(); }
```

Comment:
nitpick: no need to store document `privately`

Reply:
Why would I make it public? It's an internal thing, it's not part of the API. Otherwise this removes the whole purpose of having visibility specifiers in the typing system.